libvcx/src/disclosed_proof.rs

```diff
@@ -787,7 +773,7 @@ mod tests {
             tails_file: None,
             timestamp: None,
         }];
-        assert_eq!(build_cred_def_json(&credential_ids).unwrap_err().kind(), VcxErrorKind::InvalidProofCredentialData);
```

Comment:
I think the tests asserting methods from `proof_utils.rs` should be in that file